src/shogun/io/streaming/InputParser.h

```diff
@@ -534,6 +534,7 @@ template <class T> void* CInputParser<T>::main_parse_loop(void* params)
 		pthread_testcancel();
 
 		current_example = examples_ring->get_free_example();
+		current_example->init_fv();
 		current_feature_vector = current_example->fv;
```

Comment:
Note: without calling `init_fv()`. `current_example->fv` is a NULL pointer, which is a bug.